### PR TITLE
JAV-318 added switch for tracing

### PR DESCRIPTION
**Pull request by eric-lee-ltk**
Signed-off-by: Eric Lee <lidagang1@huawei.com>

### CODE REVIEW

File: integration-tests/spring-zuul-tracing-tests/src/test/java/io/servicecomb/spring/cloud/zuul/tracing/SpringCloudZuulTracingDisableTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import io.servicecomb.foundation.common.utils.BeanUtils;
+
+@DirtiesContext
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = TracedZuulMain.class, webEnvironment = RANDOM_PORT)
+public class SpringCloudZuulTracingDisableTest {
+
+  @BeforeClass
+  public static void init() throws Exception {
+    System.setProperty(CONFIG_TRACING_ENABLED_KEY, "false");
```
Comment (WillemJiang) [inline integration-tests/spring-zuul-tracing-tests/src/test/java/io/servicecomb/spring/cloud/zuul/tracing/SpringCloudZuulTracingDisableTest.java]: Please make sure you clean up the system property once you finish the test.

**Comment by WillemJiang** [comment]
@eric-lee-ltk Please take a look at the failed PR.

**Comment by eric-lee-ltk** [comment]
@WillemJiang The failed log can be resolved by adding the setUpLocalRegistry. However, test classes annotates with SpringBootTest under the same package share the context, which makes the second test class fail. Besides, there are few documents about the usage of SpringBootTest annotation in this case. If we do not use the SpringBootTest annotation like the previous spring-springmvc-tests section, then the autowired TestRestTemplate will become invalid and we have to find another way to get the zuul's running port.

**Comment by WillemJiang** [comment]
Maybe you can setup the test without tracing enable on the other package to workaround this issue.

**Pull request merged**